README.md

```diff
@@ -9,6 +9,11 @@ Cloudfuse supports clouds with an S3 or Azure interface.
 ## Installation
 Download the provided installation packages for your preferred operating system.  -- COMING SOON!
 
+### Windows
+
+ Download and run (as administrator) the [.exe installer](https://github.com/Seagate/cloudfuse/releases/download/0.2.0/cloudfuse_install_Windows_x86_64.exe) from our the latest [release](https://github.com/Seagate/cloudfuse/releases). Uncheck the "Launch Cloudfuse" upon finishing the installation. Run the GUI separately as admin after the install completes.

```

Comment:
Could you change the download link to https://github.com/Seagate/cloudfuse/releases. That link is specific to the 0.2.0 release and we would need to update this each time. Probably easier to just point them to the releases and ask them to download the Windows exe.